docs/user-guide/catalog.md

```diff
@@ -218,6 +225,11 @@ Enabling that option, all layers added to the map from this catalog source will
 
 * *Layer tile size*: it represents tile size (width and height) to be used for tiles of all layers added to the map from the catalog source (`256x256` or `512x512`). For layers already loaded on the map, it is possible to change the tile size through the [Layer Settings](https://mapstore.readthedocs.io/en/latest/user-guide/layer-settings/#display) tool as usual.
 
+* *Server Type*: to specify the type of server the entered WMS service url comes from. The options are two: `Geoserver` or `No Vendor`which can be mapproxy, mapserver or other.
```

Comment:
* *Server Type*: to specify the server type of the specified WMS service URL. Possible options are two: `Geoserver` or `No Vendor` which can be for example MapProxy, MapServer or other.